src/app/onprem/license-nag-message-management/nag-message.styl

```diff
@@ -0,0 +1,12 @@
+.nag-message
+  top 0
+  left 0
+  right 0
+  position fixed
+  z-index 16777271
+  line-height 30px
+  text-align center
+  background-color #FAC090
```

Comment:
is that this color could be moved at upper level ? and maybe we already have one matching